src/filesystem/impls/filer/lib/handlers.js

```diff
@@ -3,10 +3,11 @@
 define(function (require, exports, module) {
     "use strict";
 
-    var Content = require("filesystem/impls/filer/lib/content");
-    var HTMLRewriter = require("filesystem/impls/filer/lib/HTMLRewriter");
-    var CSSRewriter = require("filesystem/impls/filer/lib/CSSRewriter");
-    var Path  = require("filesystem/impls/filer/BracketsFiler").Path;
```

Comment:
OK, I'll revert the change and rebase with the new brackets-browser-livedev update.